conf/messages

```diff
@@ -227,3 +227,5 @@ time.invalidFormat=Time couldn''t get parsed.
 point.invalidFormat=Invalid format. Use something like: 4, 63, 24
 
 mturk.qualification.invalid=The entered qualification is unknown
+
+script.norFound=Task couldn''t be found
```

Comment:
Maybe notFound instead of norFound? :) and we’re not talking about tasks but scripts, here, right?

Reply:
Maybe ;)
Will do